spec/controllers/api/v1/tags_controller_spec.rb

```diff
@@ -99,7 +99,7 @@
 
     [:setting, :label, :content_warning].each do |type|
       it "should support getting #{type} tags" do
-        tag = create(type)
+        tag = create(type) # rubocop:disable Rails/SaveBang
```

Comment:
Vaguely don't like the look of this inline :P

It's annoying it detects this incorrectly, and I think it's a bug.

Reply:
I tried writing it as `FactoryBot.create(type)` but that didn't fix it

Reply:
Yeah, I'm mostly just surprised that it recognizes it without a model in front of it. I'm okay leaving it as is for now.